Cargo.toml

```diff
@@ -13,14 +13,23 @@ keywords = ["integer", "u256", "ethereum"]
 categories = ["cryptography::cryptocurrencies", "mathematics", "no-std"]
 
 [workspace]
-members = [
-  "bench",
-  "fuzz",
-  "intrinsics",
-]
+members = ["bench", "fuzz", "intrinsics"]
 
 [features]
+alloc = []
 llvm-intrinsics = ["ethnum-intrinsics"]
+serde = ["crate-serde"]
+scale = ["parity-scale-codec"]
+std = ["alloc"]
 
 [dependencies]
 ethnum-intrinsics = { version = "1", path = "intrinsics", optional = true }
+rlp = { version = "0.5", optional = true }
+crate-serde = { package = "serde", version = "1", optional = true }
+parity-scale-codec = { version = "3", features = [
+    "max-encoded-len",
+], optional = true }
+
+[dev-dependencies]
+crate-serde = { package = "serde", version = "1", features = ["derive"] }
```

Comment:
```suggestion
```

It looks like neither `#[derive(Serialize)]` nor `#[derive(Deserialize)]` are used so this can probably be removed.